js/codigo.js

```diff
@@ -0,0 +1,35 @@
+
+                    //1 is rock, 2 is paper , 3 is Scissors 
+                    let jugador =0
```

Comment:
Change variable name to English.
Review format document conventions according to the following document:  https://google.github.io/styleguide/jsguide.html